science/netcdf-fortran/Portfile

```diff
@@ -19,7 +19,7 @@ PortGroup                   github 1.0
 #        I/O feature disabled. Abort.
 mpi.enforce_variant         netcdf
 
-github.setup                Unidata netcdf-fortran 4.5.2 v
+github.setup                Unidata netcdf-fortran 4.5.3 v
 revision                    2
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Thanks, forgot. Done